x-pack/plugin/core/src/main/resources/logs-default-policy.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
We should discuss more what our default ILM policy should be moving forward in the context of the new indexing strategy. @jpountz